build/FindQt5.cmake

```diff
@@ -36,10 +36,14 @@ if (WIN32)
       )
 endif(WIN32)
 
-#find_package(Qt5Core 5.15.0 QUIET)
-#if (Qt5Core_FOUND)
-#    message(FATAL_ERROR "MuseScore 3 does not support Qt 5.15 and later (Palettes panel shows no palettes)")
-#endif()
+# For Windows, because of these lines, for some unknown reason, the build of the .msi package fails.
+if(NOT ${CMAKE_HOST_SYSTEM_NAME} MATCHES "Windows")
+    find_package(Qt5Core 5.15.0 QUIET)
+    if (Qt5Core_FOUND)
+        message(FATAL_ERROR "MuseScore 3 does not support Qt 5.15 and later (Palettes panel shows no palettes)")
```

Comment:
Could this get changed to "MuseScore 3 does not support Qt 5.15: 5.15.0 shows empty palettes panel, 5.15.1 and later crash when opening pre-3.6 scores due to QTBUG-77337"?
See https://github.com/musescore/MuseScore/commit/38cd0db5c786bdefc604ffd6748366474ece3749

Reply:
changed